src/forms.ts

```diff
@@ -30,8 +36,12 @@ export type FieldErrorCode = keyof typeof FieldErrorCodeEnum;
 
 export interface FormError {
   field?: string;
-  code?: FormErrorCode | FieldErrorCode;
+  // @TODO: provide Stripe error codes from Formspree API so we can change the string type here to StripeErrorType
+  code?: FormErrorCode | FieldErrorCode | string;
   message: string;
+  details?: {
+    stripeCode: string;
```

Comment:
stripeCode should also be optional I think. There could be a plugin in the future that uses a different key

Reply:
True